### ISSUE TITLE
How to use `opts`?

### ISSUE THREAD

**Issue by enguerran** [Teamwork/node-auto-launch#88]
![sample code](https://user-images.githubusercontent.com/701648/46484660-b606b380-c7e9-11e8-8739-f88cb0c02214.png)

In one of our projects, [Cozy Drive for Desktop](https://github.com/cozy-labs/cozy-desktop), we use the library to manage the creation and suppression of `~/.config/autostart/{appName}.desktop`.

The executable is named through our build process that uses `electron-builder`, here is a `.desktop` sample that is created:

```
➜  cozy-desktop (master) ✗ cat ~/.local/share/applications/appimagekit-cozydrive.desktop
[Desktop Entry]
Name=Cozy Drive
Comment=Save them safely in your open source personal cloud, access them anywhere, anytime with the mobile application and share them with the world or just your friends and colleagues. You can host your own Cozy Cloud, and or use the hosting services. Your freedom to chose is why you can trust us.
Exec="/home/enguerran/Development/cozy/cozy-desktop/releases/CozyDrive-3.9.1-beta.2-x86_64.AppImage" %U
Terminal=false
Type=Application
Icon=appimagekit-cozydrive
X-AppImage-Version=3.9.1-beta.2.5865
X-AppImage-BuildId=34f00d20-bdce-11a8-0949-6f597858ee2b
StartupNotify=true
Categories=Network;FileTransfer;
X-Desktop-File-Install-Version=0.24
X-AppImage-Comment=Generated by /tmp/.mount_CozyDrPcYa5G/AppRun
TryExec=/home/enguerran/Development/cozy/cozy-desktop/releases/CozyDrive-3.9.1-beta.2-x86_64.AppImage
```

As you can see, the `Exec` value is leading to a binary AppImage named `CozyDrive-3.9.1-beta.2-x86_64.AppImage`.

Long story short: given the following `opts` value to `auto-launch` we were expecting to see `~/.config/autostart/Cozy-Desktop.desktop`:

https://github.com/cozy-labs/cozy-desktop/blob/12e6d212c8b810542a33f36283b75a1af9819b08/gui/js/autolaunch.js#L3-L6

```
const opts = {
  name: 'Cozy-Desktop',
  isHidden: true
}
```

But as you can see in the screenshot, with opts set to
```
const opts = {
  name: 'cozydrive',
  path: '/usr/bin/ls'
}
```

We get `~/.config/autostart/ls.desktop`. So `opts.name` seems override by `fixOpts` and `execPath`.

Is that a wanted behavior?

**Comment by adam-lynch** [comment]
Please see that we're looking for contributors / maintainers: #64.

I'm happy to give access to people who are willing to help improve things, merge pull-requests, close issues, etc.

**Comment by enguerran** [comment]
Sorry, not on the project anymore.
